react/features/video-layout/functions.js

```diff
@@ -73,7 +73,8 @@ export function shouldDisplayTileView(state: Object = {}) {
     return Boolean(
         state['features/video-layout']
             && state['features/video-layout'].tileViewEnabled
-            && !state['features/etherpad'].editing
+            && (!state['features/etherpad']
```

Comment:
I can't wait until we update RN to 0.56, which uses Babel 7 so we can use optional chaining: https://babeljs.io/docs/en/next/babel-plugin-proposal-optional-chaining.html